### DISCUSSION TITLE
Docs Improvement General

### DISCUSSION THREAD

**Discussion by c-o-i-n-w-i-t-c-h** [elizaOS/eliza#499] [General]
Please drop any comments about how we can improve the docs here!
https://ai16z.github.io/eliza/docs/intro/

**Comment by c-o-i-n-w-i-t-c-h** [comment]
Add a note for linux users: only debian/ubuntu are supported due to Playright dependencies

**Comment by c-o-i-n-w-i-t-c-h** [comment]
Walkthrough suggestion:

# install node
1. install nvm (replace bash with your shell if necessary)
```
$ curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.1/install.sh | bash
$ source ~/.bashrc
```
2. install the supported nodejs version
```
$ nvm install 23.1
$ nvm use 23.1
```
3. install pnpm
```
$ npm install -g pnpm
$ pnpm --version
```

**Comment by c-o-i-n-w-i-t-c-h** [comment]
The version of eliza to `git checkout` is outdated on https://ai16z.github.io/eliza/docs/quickstart/

How can we automatically update this in the docs whenever a new stable version drops?

**Comment by metakai1** [comment]
Debugging tips for noobs:
- logs location
- how to turn on debugging for various features

**Comment by dfischer** [comment]
Common deployment patterns for production (paas, vpn, w/e).
Fine tuning loop (use data from conversations to fine tune or context stuff dynamically over time so it's automatically improving or optimizing and most interacted with / successful things).
